inference-engine/tests/functional/plugin/shared/src/single_layer_tests/pooling.cpp

```diff
@@ -95,7 +135,48 @@ void PoolingLayerTest::SetUp() {
     function = std::make_shared<ngraph::Function>(results, params, "pooling");
 }
 
+void GlobalPoolingLayerTest::SetUp() {
+    poolSpecificParams poolParams;
+    InferenceEngine::Precision netPrecision;
+    size_t channels;
+    std::tie(poolParams, netPrecision, channels, targetDevice) = this->GetParam();
+    ngraph::helpers::PoolingTypes poolType;
+    std::vector<size_t> kernel, stride;
+    std::vector<size_t> padBegin, padEnd;
+    ngraph::op::PadType padType;
+    ngraph::op::RoundingType roundingType;
+    bool excludePad;
+    std::tie(poolType, kernel, stride, padBegin, padEnd, roundingType, padType, excludePad) = poolParams;
+
+    std::vector<size_t> inputShape = {1, channels, kernel[1], kernel[0]};
+
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    auto paramOuts = ngraph::helpers::convert2OutputVector(
+            ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+    std::shared_ptr<ngraph::Node> pooling;
+    switch (poolType) {
+        case ngraph::helpers::PoolingTypes::MAX:
+            pooling = std::make_shared<ngraph::opset1::MaxPool>(paramOuts[0], stride, padBegin, padEnd, kernel,
+                                                                roundingType,
+                                                                padType);
+            break;
+        case ngraph::helpers::PoolingTypes::AVG:
+            pooling = std::make_shared<ngraph::opset1::AvgPool>(paramOuts[0], stride, padBegin, padEnd, kernel,
```

Comment:
As I know it's better to use the last opset version which is now `opset3`,

Reply:
Fixed